src/sentry/web/frontend/accounts.py

```diff
@@ -134,8 +136,14 @@ def start_confirm_email(request):
             status=429,
         )
 
-    has_unverified_emails = request.user.has_unverified_emails()
-    if has_unverified_emails:
+    if 'primary-email' in request.POST:
+        email = request.POST.get('email')
+        email_to_send = UserEmail.objects.get(user=request.user, email=email)
```

Comment:
I guarantee at some point, this will end up raising a 500 for `UserEmail.DoesNotExist` :) You can choose to handle it now, but I won't block because of it. It'll just be noisy in Sentry.